src/cmd/tools/m3ctl/main/apply/cmd.go

```diff
@@ -0,0 +1,108 @@
+package apply
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/m3db/m3/src/cmd/tools/m3ctl/main/errors"
+	"github.com/m3db/m3/src/cmd/tools/m3ctl/main/globalopts"
+	"github.com/m3db/m3/src/x/config/configflag"
+)
+
+type applyVals struct {
+	yamlFlag *configflag.FlagStringSlice
+}
+type Context struct {
```

Comment:
context usually means Go context.Context, this sounds should be something like 'options'

Reply:
got rid of all that. put in cobra.